core/src/com/unciv/models/ruleset/tile/TileImprovement.kt

```diff
@@ -32,7 +32,13 @@ class TileImprovement : NamedStats() {
 
     fun getDescription(ruleset: Ruleset): String {
         val stringBuilder = StringBuilder()
-        if (this.clone().toString().isNotEmpty()) stringBuilder.appendln(this.clone().toString())
+        val statsDesc = this.clone().toString()
```

Comment:
The comment is not related to _your_ changes, however, it is a quite strange usage (and implementation!) of the `clone()` method.
That was a surprise for me, that `clone()` does not a clone but a partial copy of the object (e.g. `name` is not copied). I would rather name it as "getStats" rather than "clone". 
But that's a quite core thing, so I do not think you should change that right now.

Reply:
I noticed that. That's why I introduced the val to name it a bit clearer (and to avoid the double calls).